src/commands/base.ts

```diff
@@ -304,6 +315,14 @@ export default abstract class Base extends Command {
                 await this.updateUserConfig({ project: this.projectKey })
             }
         }
+
+        const project = projects.find((proj) => proj.key === this.projectKey)
```

Comment:
can maybe make finding a project a util function

Reply:
to me it seems like too simple of a call to make it a separate func

Reply:
can combine it with the below call that throws, or just make the error message a constant so it's not repeated

Reply:
updated lmk what you think